Include/cpython/pyctype.h

```diff
@@ -29,5 +33,9 @@ PyAPI_DATA(const unsigned char) _Py_ctype_toupper[256];
 #define Py_TOLOWER(c) (_Py_ctype_tolower[Py_CHARMASK(c)])
 #define Py_TOUPPER(c) (_Py_ctype_toupper[Py_CHARMASK(c)])
 
+#ifdef __cplusplus
+}
+#endif
+
```

Comment:
and remove this one :-)

To be consistent with other header files.

Reply:
Done!